packages/grant-explorer/src/features/discovery/ExploreRoundsPage.tsx

```diff
@@ -18,14 +18,17 @@ const ExploreRoundsPage = () => {
   // Pass the filter from the search params and build the graphql query
   const rounds = useFilterRounds(filter, getEnabledChains());
 
+  const publicRounds = rounds.data?.filter(round => round.roundMetadata.roundType?.toLowerCase() !== "private");
```

Comment:
this is potentially an expensive calculation that we're running on each render, (filtering up to 500 rounds) will slow things down for users considerably: https://outshift.cisco.com/blog/avoiding-expensive-recalculation-in-react-with-the-usememo-hook